README.md

```diff
@@ -4,27 +4,10 @@
 
 This guide is meant for students and employees of the complex systems modelling group at UiT. External collaborators or people with interest in the groups work are also welcome to use and contribute to the provided repositories. 
 
-- [Repositories](#repos)
-    - [`superposed-pulses`](#superposed-pulses)
-    - [`2d-propagating-blobs`](#2d-propagating-blobs)
-    - [`fpp-analysis-tools`](#fpp-analysis-tools)
-    - [`fpp-closed-expressions`](#fpp-closed-expressions)
-    - [`fpp-sle`](#fpp-sle)
-    - [`cosmoplots`](#cosmoplots)
-    - [`xblobs`](#xblobs)
-    - [`Complexity-Entropy-Analysis`](#Complexity-Entropy-Analysis)
-    - [`ebm`](#ebm)
-    - [`SOFTX-D-16-00023`](#SOFTX-D-16-00023)
-    - [`uit_sandpiles`](#uit_sandpiles)
-    - [`poster-template`](#poster-template)
-    - [`PhD-thesis`](#PhD-thesis)
-
-- [Publication specific repositories](#pub-repos)
-    - [`dirac-comb-and-exponential-frequency-spectra-in-nonlinear-dynamics`](#dirac) 
-- [Contributing to uit-cosmo ](#contributing)
-
-
 ## Repositories<a name="repos"></a>
+<details>
+	<summary> <strong> superposed-pulses </strong> </summary>	
```

Comment:
Something like the below would make the GitHub menu work I think:

```html
<summary><strong><h2><code>superposed-pulses</code></h2></strong></summary>
```

Or maybe without `<code></code>` looks cleaner?

```html
<summary><strong><h2>superposed-pulses</h2></strong></summary>
```

Reply:
I personally find <h2> to big since not all repos would fit on one screen anymore. I don't think that the GitHub menu is a concern since in the current commit all repos are compactly together. But please feel free to change the code in this PR if you think otherwise!

Reply:
For sure, h2 was just to put something there instead of ##.

What do you think removing the markdown headings and just link to the repos?

Reply:
Made a quick suggestion so the broken links in the github menu don't show in the [comment-block](https://github.com/uit-cosmo/user-guide/tree/comment-block) branch. But I'm happy with whatever you go with:)

Reply:
Thx! I honestly don't mind which version we use. If you create a PR from the comment-block branch I can merge that one.